pyinfraformat/core/coord_utils.py

```diff
@@ -192,14 +194,82 @@ def check_area(holes, country="FI"):
         raise ValueError("holes -parameter is unkown input type")
 
 
-def project_hole(hole, output_epsg="EPSG:4326"):
+def get_n43_n60_points():
+    """Get MML reference points for height systems n43-n60."""
+    url = (
+        r"https://www.maanmittauslaitos.fi/sites/maanmittauslaitos.fi/files/attachments/2019"
+        + r"/02/n43n60triangulationVertices.txt"
+    )
+    df = pd.read_csv(url, delim_whitespace=True, header=None)
+    df.columns = ["Point", "X", "Y", "diff", "Karttalehden numero"]
+    df = df.loc[:, ["X", "Y", "diff"]]
+    return df
+
+
+def get_n60_n2000_points():
+    """Get MML reference points for height systems n60-n2000."""
+    url = (
+        r"https://www.maanmittauslaitos.fi/sites/maanmittauslaitos.fi/files/attachments/2019"
+        + r"/02/n60n2000triangulationVertices.txt"
+    )
+    df = pd.read_csv(url, encoding="latin-1", delim_whitespace=True, header=None)
+    df.columns = ["Point", "X", "Y", "N60", "N2000"]
+    df["diff"] = df["N2000"] - df["N60"]
+    df = df.loc[:, ["X", "Y", "diff"]]
+    return df
+
+
+def get_closest(df, point):
+    """Get closest row in df for point (x, y)."""
+    point = np.array(point)[:, None].T
+    dists = distance.cdist(point, df.loc[:, ["X", "Y"]])
```

Comment:
Euclidian space is ok for here? (pyproj has possibility for dist too)

Reply:
Different implementation. Nearest -> linear interpolation